containers-contrib-readonly/src/Map/Contrib/Bench.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Map.Contrib.Bench where
+
+import Map
+import Gauge.Main (Benchmark, bench, defaultMain, whnf)
+import Relude hiding (fromList, Map)
+
+benchmark
```

Comment:
Let's rename this `benchmark` in a way that it tells about what it does, like `benchmarkSequentialLookup`.

Reply:
@yigitozkavci Hmm, if this is the function that runs all benchmarks than okay